plugin/emacs/flycheck-quicklintjs.el

```diff
@@ -50,18 +50,22 @@
 
 https://quick-lint-js.com"
   :command	("quick-lint-js"
-			 "--output-format=gnu-like"
-			 "--stdin"
-			 (eval flycheck-quicklintjs-args))
+                 "--output-format=gnu-like"
+                 "--stdin"
+                 (eval flycheck-quicklintjs-args))
   :standard-input t
   :error-patterns ((error
                     line-start "<stdin>:" line ":" column ":"
                     (zero-or-more whitespace) "error:" (zero-or-more whitespace)
-                    (message) line-end)
+                    (message) "[" (id "E" (one-or-more digit)) "]" line-end)
                    (warning
                     line-start "<stdin>:" line ":" column ":"
                     (zero-or-more whitespace) "warning:" (zero-or-more whitespace)
-                    (message) line-end))
+                    (message) "[" (id "E" (one-or-more digit)) "]" line-end))
+  :error-explainer (lambda (err)
+                     (let ((error-code (flycheck-error-id err))
+                           (url "https://quick-lint-js.com/errors/#%s"))
+                       (and error-code `(url . ,(format url error-code)))))
```

Comment:
How would you pronounce this expression aloud? =S (I'm an Elisp newb.)

Reply:
"error-code and quote to cons url onto string" perhaps? not sure if that helps much, but it's somehow equialent to:
```lisp
(and error-code (cons url (format url error-code)))
```
the backtick quoting differs from apostrophe in that it allows some evaluation inside the quoting, here `,(format url error-code)` becomes a cons which car is `https://quick-lint-js.com/errors/#%s` and cdr is `https://quick-lint-js.com/errors/#ENNN`